chia/farmer/farmer_api.py

```diff
@@ -321,13 +324,13 @@ async def respond_signatures(self, response: harvester_protocol.RespondSignature
         assert pospace is not None
         include_taproot: bool = pospace.pool_contract_puzzle_hash is not None
 
-        # we skip plot filter validation here, we trust the harvester
         computed_quality_string = verify_and_get_quality_string(
             pospace,
             self.farmer.constants,
             response.challenge_hash,
             response.sp_hash,
             height=None,
+            filter_prefix_bits=response.filter_prefix_bits,
```

Comment:
would it not be possible to look up the filter size from `self.farmer.quality_str_to_identifiers` instead?
I suppose we don't technically have the quality string before this call, but it seems we already store the filter size